content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -69,23 +69,30 @@ See the Google Kubernetes Engine (GKE) guide to
 
 ## Deploying Kubeflow Pipelines
 
-1. Deploy the latest version of Kubeflow Pipelines:
-
-     **Note**: The following commands apply to Kubeflow Pipelines version 0.2.0 and later.
+1. Deploy the Kubeflow Pipelines:
 
      ```
      export PIPELINE_VERSION={{% pipelines/latest-version %}}
-     kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/base/crds?ref=$PIPELINE_VERSION
+     kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION
      kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
      kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
      ```
 
      The Kubeflow Pipelines deployment requires approximately 3 minutes to complete.
 
-     **Note**: To deploy versions of Kubeflow Pipelines prior to version 0.2.0, run:
+     **Note**: The above commands apply to Kubeflow Pipelines version 0.4.0 and later.
 
+     For KFP version 0.2.0 ~ 0.3.0, use:
      ```
-     export PIPELINE_VERSION=<kfp-version>
+     export PIPELINE_VERSION=<kfp-version-between-0.2.0-and-0.3.0>
+     kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/base/crds?ref=$PIPELINE_VERSION
+     kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
+     kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+     ```
+
+     For KFP version < 0.2.0, use:
```

Comment:
My 2c: perhaps we can also archive the installation instruction for those stale version and put a link to somewhere in Github repo here, just to prevent this page from being very nested.

Reply:
Yes, I agree. Do you think we need to do it now?

Reply:
No. Feel free to do that in future.